src/AutoRest.CSharp.V3/Output/Models/ClientBase.cs

```diff
@@ -16,14 +16,12 @@ internal abstract class ClientBase: TypeProvider
         private const string OperationsSuffixValue = "Operations";
         protected string ClientSuffix { get; }
 
-        private readonly BuildContext _context;
         private readonly TypeFactory _typeFactory;
 
         protected ClientBase(BuildContext context): base(context)
         {
             ClientSuffix = context.Configuration.AzureArm ? OperationsSuffixValue : ClientSuffixValue;
             _typeFactory = context.TypeFactory;
-            _context = context;
```

Comment:
Was this removed because it is also in the base class?

Reply:
yeah, made it protected in the baseclass